code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -61,20 +45,7 @@
 	open_door_sprite = "hop_door"
 
 /obj/structure/closet/secure_closet/hop2/populate_contents()
```

Comment:
This locker kinda lost its purpose now but I don't want this to be a mapping PR to remove them.

Reply:
you can just leave it be if it would put you in the mapping queue tbh, you're always welcome to open up a second simpler PR if think this should be removed.